src/definition/mod.rs

```diff
@@ -262,11 +264,109 @@ pub enum ARIAPropertyDefinitionType {
 }
 
 pub struct ARIAPropertyDefinition {
+    pub key: ARIAProperty,
     pub type_: ARIAPropertyDefinitionType,
-    pub values: Option<Vec<String>>,
+    pub values: Option<&'static [&'static str]>,
     pub allow_undefined: Option<bool>,
 }
 
+impl Serialize for ARIAPropertyDefinition {
```

Comment:
this serializer implementation is just for snapshot testing. I wondered whether should I split this with the feature flag, but I thought it will not be a problem that this crates includes the implementation for the testing

Reply:
#[cfg(debug_assertions)]

should do the trick

Reply:
that is indeed the case...thanks!! I use #cfg[test] for this implementation of serializer because this implementation is needed only for testing https://github.com/oxc-project/aria-query/pull/8